src/main/java/liquibase/ext/spanner/sqlgenerator/AddDefaultValueGeneratorSpanner.java

```diff
@@ -16,23 +16,44 @@
 import liquibase.database.Database;
 import liquibase.exception.ValidationErrors;
 import liquibase.ext.spanner.ICloudSpanner;
+import liquibase.sql.Sql;
+import liquibase.sql.UnparsedSql;
 import liquibase.sqlgenerator.SqlGenerator;
 import liquibase.sqlgenerator.SqlGeneratorChain;
 import liquibase.sqlgenerator.core.AddDefaultValueGenerator;
+import liquibase.statement.DatabaseFunction;
 import liquibase.statement.core.AddDefaultValueStatement;
 
 public class AddDefaultValueGeneratorSpanner extends AddDefaultValueGenerator {
   static final String ADD_DEFAULT_VALUE_VALIDATION_ERROR =
-      "Cloud Spanner does not support adding a default value to a column";
+      "Cloud Spanner supports default values for columns, but only with built-in functions such as CURRENT_TIMESTAMP() and GENERATE_UUID()";
 
   @Override
   public ValidationErrors validate(
       AddDefaultValueStatement statement, Database database, SqlGeneratorChain sqlGeneratorChain) {
     ValidationErrors errors = super.validate(statement, database, sqlGeneratorChain);
+    if (statement.getDefaultValue() instanceof DatabaseFunction) {
+      return errors;
+    }
     errors.addError(ADD_DEFAULT_VALUE_VALIDATION_ERROR);
     return errors;
   }
 
+  @Override
+  public Sql[] generateSql(final AddDefaultValueStatement statement, final Database database, SqlGeneratorChain sqlGeneratorChain) {
+    Object defaultValue = statement.getDefaultValue();
+    StringBuilder queryStringBuilder = new StringBuilder();
+    queryStringBuilder.append("ALTER TABLE ");
+    queryStringBuilder.append(database.escapeSequenceName(statement.getCatalogName(), statement.getSchemaName(), statement.getTableName()));
```

Comment:
nit: is there an `escapeTableName(..)` method that we can use instead? (It probably does not make any difference, but this looks a bit weird)